public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -552,19 +552,26 @@ export const Explorer = ({
                         <EuiSpacer size="m" />
                       </>
                     )}
-                    <DataGrid
-                      http={http}
-                      pplService={pplService}
-                      rows={explorerData.jsonData}
-                      rowsAll={explorerData.jsonDataAll}
-                      explorerFields={explorerFields}
-                      timeStampField={queryRef.current![SELECTED_TIMESTAMP]}
-                      rawQuery={appBasedRef.current || queryRef.current![RAW_QUERY]}
-                      totalHits={_.sum(countDistribution.data?.['count()'])}
-                      requestParams={requestParams}
-                      startTime={appLogEvents ? startTime : dateRange[0]}
-                      endTime={appLogEvents ? endTime : dateRange[1]}
-                    />
+                    {((countDistribution.data && countDistribution.data['count()']) ||
```

Comment:
it's little bit hard to understand what are the conditions to render datagrid. I think no matter what is the condition, as long as there are datarows, we should show data grid, it is not supposed to base on cound distribution or what type of datasource